contracts/database/orm/factory.go

```diff
@@ -0,0 +1,12 @@
+package orm
+
+//go:generate mockery --name=Factory
+type Factory interface {
+	Count(count int) Factory
+	Raw() any
```

Comment:
S: This method seems to repeat with `Make`. We don't need to return a map like Laravel.

Reply:
done